### ISSUE TITLE
[BUG] Generated methods are not considered in javadoc task

### ISSUE THREAD

**Issue by burdoto** [rzwitserloot/lombok#2263]
**Describe the bug**
The methods generated by lombok are not considered by the gradle `javadoc` task.

**To Reproduce**
See this setup: https://github.com/codebottle-io/codebottle-java/tree/b943a68a1a172f562c94ebe3365a694523a78af7

**Expected behavior**
At least, empty JavaDocs should be generated for the methods.
Maybe we can write JavaDoc in the source code and annotate that so it will be used for the generated members?

**Version info (please complete the following information):**
 - Lombok v`1.18.10`
 - IntelliJ, Gradle

**Additional context**
`#getToken` getter:
> In my example, I implemented `getToken` myself due to it returning `Optional<String>` instead of `@Nullabe String`, which I cannot define in lombok, so I already added it in the sourcecode.
- Source: https://github.com/codebottle-io/codebottle-java/blob/b943a68a1a172f562c94ebe3365a694523a78af7/src/main/java/io/codebottle/api/CodeBottle.java#L55-L57
- JavaDoc: https://codebottle-io.github.io/codebottle-java/io/codebottle/api/CodeBottle.html#getToken--

`#getHttpClient` getter:
> The required `#getHttpClient` getter I let lombok generate since there is nothing to take into account here.
- Source: https://github.com/codebottle-io/codebottle-java/blob/b943a68a1a172f562c94ebe3365a694523a78af7/src/main/java/io/codebottle/api/CodeBottle.java#L33-L34
- Theoretical JavaDoc link (unsatisfied): https://codebottle-io.github.io/codebottle-java/io/codebottle/api/CodeBottle.html#getHttpClient--
> The JavaDoc of this method is not existent.

Used for generating the JavaDocs is this Gradle task configuration:
```groovy
javadoc {
    source = sourceSets.main.java
    options {
        encoding = 'UTF-8'
        destinationDirectory(file("./docs/"))
        links = [
                "https://docs.oracle.com/javase/8/docs/api/",
                "https://docs.oracle.com/javaee/7/api/"
        ]
    }
}
```

**Comment by rzwitserloot** [comment]
I do not understand what is going wrong and what you want to occur. I am not a gradle user, so some elaboration may be required.

**Comment by Maaartinus** [comment]
> The methods generated by lombok are not considered by the gradle javadoc task.

AFAIK that's a known unsolvable problem. You have to run delombok and then generate the javadoc from it.

**Comment by burdoto** [comment]
Closed, related to #2265.
